wolfcrypt/src/ecc.c

```diff
@@ -14519,6 +14530,44 @@ int wc_X963_KDF(enum wc_HashType type, const byte* secret, word32 secretSz,
 }
 #endif /* HAVE_X963_KDF */
 
+#ifdef WOLFSSL_SE050
+/* Use specified hardware key ID with ecc_key operations. Unlike devId,
+ * keyId is a word32, can be used for key IDs larger than an int.
+ *
+ * key    initialized ecc_key struct
+ * keyId  hardware key ID which stores ECC key
+ * flags  optional flags, currently unused
+ *
+ * Return 0 on success, negative on error */
+int wc_ecc_use_key_id(ecc_key* key, word32 keyId, word32 flags)
```

Comment:
Did you consider leveraging `wc_ecc_init_id`?

Reply:
Had considered, but didn't seem like a good fit since args were for devId (int) and id (unsigned char*).  When really needed to set the SE050 `word32 keyId`.